### PR TITLE
Fix RMS calculation in mictest.js

### PR DESCRIPTION
**Pull request by fsolenberg**
Calculate the Root Mean Square, as opposed to Root Absolute Mean. Also, getting to know the GitHub workflow.

### CODE REVIEW

File: src/js/mictest.js [hunk 1]
```diff
@@ -128,8 +128,10 @@ MicTest.prototype = {
   testInputVolume: function(buffer, channel) {
     var data = buffer.getChannelData(channel);
     var sum = 0;
+    var s = 0;
```
Comment (KaptenJansson) [inline src/js/mictest.js]: Can you rename it to something meaningful? Maybe replace "sample" with "i" and "s" with "sample"?

**Comment by KaptenJansson** [comment]
Also you need to sign the CLA or use your google email for your git checkout (needs to be added to your github account), do git config -user.email foo@google.com in the root of the testrtc checkout.

File: src/js/mictest.js (hunk 1 above)
Comment (fsolenberg) [inline src/js/mictest.js]: My pleasure!

**Comment by KaptenJansson** [comment]
LGTM Thank for your fixes!

**Pull request merged**